src/pyggdrasil/serialize/_to_json.py

```diff
@@ -91,3 +113,78 @@ def generate_node(node_dict: dict, parent: Optional[TreeNode]) -> TreeNode:
         return new_node
 
     return generate_node(dct, parent=None)
+
+
+def save_tree_node(Tree: TreeNode, output_fp: Path):
+    """Saves Tree object as dict /json to disk.
+
+    Args:
+        tree: Tree object to be saved
+        output_dir: directory to save tree to
+    Returns:
+        None
+    """
+
+    tree_node = serialize_tree_to_dict(Tree, serialize_data=lambda x: x)
+
+    with open(output_fp, "w") as f:
+        json.dump(tree_node, f, cls=JnpEncoder)
+
+
+def read_tree_node(output_fp: Path):
+    """Reads Json file to Tree object from disk.
+
+    Args:
+        output_fp: directory to save tree to
+    Returns:
+        None
+    """
+
+    with open(output_fp, "r") as f:
+        tree_node = json.load(f)
+
+    return deserialize_tree_from_dict(tree_node, deserialize_data=lambda x: x)
+
+
+def save_mcmc_sample(sample: MCMCSample, output_dir: Path, **kwargs) -> None:
```

Comment:
Why can't `**kwargs` be replaced with a simple `timestamp: Optional[str] = None`?